pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/RuntimeASTTransformer.groovy

```diff
@@ -700,6 +700,47 @@ class RuntimeASTTransformer {
         return constX(null)
     }
 
+    /**
+     * Generates the AST (to be CPS-transformed) for instantiating a list of {@link ParallelContent}.
+     *
+     * @param original The parsed AST model of a stage
+     * @return The AST for a list of {@link ParallelContent}, or the constant null expression if the original
+     * cannot be transformed.
+     */
+    Expression transformParallelContent(@CheckForNull ModelASTStage original) {
+        if (isGroovyAST(original) && !original.parallelContent?.isEmpty()) {
```

Comment:
#210 was merged only 7 days ago :) 
`!original.parallelContent?.isEmpty()` can be shortened to just `!original.parallelContent` iirc "Groovy truthiness" checks for both `null` and non empty for lists.
But I could be remembering wrongly ;)

Reply:
boo, this was actually older code that I copy-pasted into an older iteration of this. Cleaning up!